percy/config.py

```diff
@@ -6,18 +6,36 @@ class AuthError(Exception):
 
 
 class Config(object):
+    
+    def __init__(self, api_url='https://percy.io/api/v1', default_widths=[], access_token=None):
```

Comment:
Style: avoid setting mutable objects as default arguments, by doing `default_widths=[]` the list is scoped outside of the method and can be accidentally mutated. Ie:

```python
>>> def foo(bar=[]):
...   bar.append(1)
...   print bar
... 
>>> foo()
[1]
>>> foo()
[1, 1]
>>> foo()
[1, 1, 1]
>>> foo()
[1, 1, 1, 1]
>>> foo()
[1, 1, 1, 1, 1]
>>> 
```

So just do `=None` instead and handle the default inside the method.